apps/foundation-demo/src/app/misc/footer/footer.component.html

```diff
@@ -0,0 +1,28 @@
+<div daff-footer>
+  <div daff-list>
```

Comment:
- [ ] I think `daff-list` makes more sense as a `<ul>` instead of `<div>`?
- [ ] Add padding: 0 0 5px 0; to each list item.